src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -87,15 +87,17 @@ public static class Constants
 
       //sensitivity values below this value will be set to zero
       public const double SENSITIVITY_THRESHOLD = 1.0e-4;
-
       public const string STD_DEV_GEOMETRIC = "Geometric Standard Deviation";
       public const string STD_DEV_ARITHMETIC = "Arithmetic Standard Deviation";
       public const string AUXILIARY_TYPE = "AuxiliaryType";
       public const string FILE = "File";
       public const string SHEET = "Sheet";
-
       public const string DEFAULT_WATERMARK_TEXT = "DRAFT";
 
+      public const int RELATIVE_ITEM_FILE_SIZE_WARNING_THRESHOLD_IN_BYTES = 5_000_000; 
+      public const int RELATIVE_ITEM_MAX_FILE_SIZE_IN_BYTES = 50_000_000; 
```

Comment:
50MB max. 5 MB warning

Reply:
I would define it as 5 * 1024 * 1024 and 50 * 1024 * 1024

Reply:
yep ok. But I need to convert back then to MB when displaying to user.